### DISCUSSION TITLE
slingerplayer flv

### DISCUSSION THREAD

**Discussion by alfuxj** [GerryDazoo/SlingerPlayer#9] [General]
because i use rtmp protocol , video code is  flv , is possible to add  FLV (Flash Video) codec ?
is very hard open flux from china with http: direct to slinger server , but with rtmp:// open flux very fast all time , without problem , the problem is i have only audio , but not the video . I recoded flux to nginx rtmp server with ffmpeg , i recoded only the video , the audio flux is not recoded.
thank in advance

**Comment by GerryDazoo** [comment]
No, sorry. The code was written to just play video streams from Slinger. But it does use the ffmpeg libraries for video. did you try typing in an rtmp://  URL?

**Reply by alfuxj** [reply]
yes i open my url nginx rtmp server work only audio stream cannot decode flv stream

**Reply by alfuxj** [reply]
if you give me private email i will send you url of my rtmp server and you will may try .

**Comment by GerryDazoo** [comment]
Unfortunately SlingerPlayer won't play FLV video. Sorry.
